guac-configuration.md

```diff
@@ -0,0 +1,164 @@
+
+- **neptune-endpoint**: `localhost`
+  - **Description**: The endpoint for the Neptune database.
+  - **When to Change**: Update this to the actual endpoint if using AWS Neptune.
+
+- **neptune-port**: `8182`
+  - **Description**: The port for the Neptune database.
+  - **When to Change**: Change if your Neptune instance uses a non-default port.
+
+- **neptune-region**: `us-east-1`
+  - **Description**: The AWS region where the Neptune database is hosted.
+  - **When to Change**: Modify to match the region of your AWS Neptune instance.
+
+- **neptune-realm**: `neptune`
+  - **Description**: The realm for the Neptune database.
+  - **When to Change**: Adjust if your setup uses a different realm.
+
+## Pub/Sub Configuration
+
+- **pubsub-addr**: `nats://localhost:4222`
+  - **Description**: The address of the NATS server for pub/sub messaging.
+  - **When to Change**: Update if your NATS server is hosted elsewhere or uses a different port.
+
+- **publish-to-queue**: `true`
+  - **Description**: Whether to publish messages to the queue.
+  - **When to Change**: Set to `false` if you do not want to use queue-based messaging.
+
+## Blob Store Configuration
+
+- **blob-addr**: `file:///tmp/blobstore?no_tmp_dir=true`
+  - **Description**: The address of the blob store.
+  - **When to Change**: Change to use a different storage backend, such as AWS S3 or Google Cloud Storage (examples).
+
+## Certifier Configuration
+
+- **interval**: `20m`
+  - **Description**: The interval at which the certifier runs.
+  - **When to Change**: Adjust based on how frequently you need certification checks.
+
+- **last-scan**: `4`
+  - **Description**: The number of hours since the last scan was run. A value of `0` means the scan will run on all packages/sources.
+  - **When to Change**: Set to `0` for a full scan or adjust based on your scanning frequency needs.
+
+- **certifier-batch-size**: `60000`
+  - **Description**: The batch size for the package pagination query.
+  - **When to Change**: Modify to optimize performance based on your system's capabilities.
+
+- **certifier-latency**: `""`
+  - **Description**: Artificial latency to throttle the certifier.
+  - **When to Change**: Use to introduce delays if needed to manage load.
+
+## Deps.dev Configuration
+
+- **deps-dev-latency**: `""`
+  - **Description**: Artificial latency to throttle deps.dev.
+  - **When to Change**: Adjust if you need to manage load on deps.dev queries.
+
+## Ingestion Configuration
+
+- **add-vuln-on-ingest**: `false`
+  - **Description**: Whether to query vulnerabilities during ingestion.
+  - **When to Change**: Set to `true` if you want to automatically check for vulnerabilities during data ingestion.
+
+- **add-license-on-ingest**: `false`
+  - **Description**: Whether to query licenses during ingestion.
+  - **When to Change**: Set to `true` if you want to automatically check for licenses during data ingestion.
+
+## CSub Configuration
```

Comment:
Let's use "Collector-Subscriber" throughout unless it's the name of a setting. This makes it a little less jargon-y.